src/libs/SearchUtils.ts

```diff
@@ -453,7 +453,7 @@ function buildAmountFilterQuery(filterValues: Partial<SearchAdvancedFiltersForm>
 }
 
 function sanitizeString(str: string) {
-    const regexp = /[<>,:= ]/g;
+    const regexp = /[^A-Za-z0-9_@./#&+\-\\';,"]/g;
```

Comment:
Space is missing from the list which causes the issue of not adding the quotes if the filter value contains space.